cvat/apps/dashboard/static/dashboard/js/dashboard.js

```diff
@@ -169,12 +167,12 @@ class TaskView {
         async function onload(overlay, text) {
             try {
                 overlay.setMessage('Required data are being downloaded from the server..');
-                const imageCache = await $.get(`/api/v1/tasks/${this._id}/frames/meta`);
-                const labelsCopy = JSON.parse(JSON.stringify(this._labels));
+                const imageCache = await $.get(`/api/v1/tasks/${this._task.id}/frames/meta`);
```

Comment:
Why do we access the server directly?

Reply:
Because this functionality still isn't implemented in cvat.js.